apps/web/package.json

```diff
@@ -31,6 +31,7 @@
     "lucia": "^3.2.0",
     "monaco-editor": "^0.50.0",
     "next": "^14.3.0-canary.87",
+    "nextjs-toploader": "^1.6.12",
```

Comment:
This is just one file package. We could do our selves but is works out of the box